tests/Conversations/ConversationTest.php

```diff
@@ -248,7 +248,9 @@ public function testExtract()
         );
         $conversation->setCustomerWaitingSince($customerWaitingSince);
 
-        $this->assertArraySubset([
```

Comment:
Unfortunately this method has been deprecated so I explicitly tested first and second level array values in the extracted conversation. 

Normally I would chunk this up into separate tests entirely to move away from a single long test, but wanted to keep changes to a minimum for now.